input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,499 @@
+For more information on how AU Core relates to other implementation guides, see [Relationship with other IGs](relationship.html).
 
-Conformance in reverse is not guaranteed, i.e. a resource conforming to [International Patient Access](https://hl7.org/fhir/uv/ipa/STU1/), [International Patient Summary](https://hl7.org/fhir/uv/ips/STU1.1/), or [US Core](https://hl7.org/fhir/us/core/STU6.1/) **MAY NOT** conform to AU Core.
+### Profile comparison
+A resource may be compliant to multiple implementation guides. In addition to AU Core profiles, a FHIR resource may be compliant to profiles published in project implementation guides, other national implementation guides, or international implementation guides.
 
-<table border="1" cellspacing="0" cellpadding="0">
+Compliance refers to a resource meeting all the requirements, constraints, and standards specified in a particular FHIR profile or an implementation guide. These requirements can include mandatory elements, cardinality constraints, data types, terminology bindings, and extensions. 
+
+The table below provides a profile only comparison from AU Core to other implementation guides. It indicates whether a resource compliant with a particular AU Core profile also satisfies (i.e., is compliant with) a profile from [International Patient Access](https://hl7.org/fhir/uv/ipa/STU1/), [International Patient Summary](https://hl7.org/fhir/uv/ips/STU1.1/), or [US Core](https://hl7.org/fhir/us/core/STU7/). 
+
+Compliance in the reverse direction is not guaranteed, i.e. a resource that is compliant with International Patient Access, International Patient Summary, or US Core **MAY NOT** be compliant with AU Core.Future updates may include reverse comparisons and CapabilityStatement evaluations.
+
+Legend:
+
+<img src="green_checkmark.svg.png" width="20"/> **Compliant**: An AU Core compliant resource meets all requirements of the compared profile.
+
+<img src="orange_checkmark.svg.png" width="20"/> **Additional requirements**: An AU Core compliant resource is compatible, but additional changes may be needed to meet all requirements of the compared profile.
+
+<img src="cross_red_circle.svg.png" width="20"/> **Incompatible**: An AU Core compliant resource is incompatible with the compared profile. A resource cannot be compliant to both.
+
+<img src="minus_symbol.svg.png" width="20"/> **No equivalent profile**: When AU Core profile has no equivalent in compared implementation guides.
```

Comment:
Replace "When AU Core profile has no equivalent in compared implementation guides." with "No equivalent profile for comparison."

Reply:
Done